app/views/content/how-to-apply-for-teacher-training/ske-explained-transcript.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Subject knowledge enhancement explained video transcript
+description: Full transcript for the 'Subject knowledge enhancement' video.
+---
+
+**Speaker 1:** SKE stands for subject knowledge enhancement and it allows you to build upon your subject knowledge that you would be expected to have as a teacher. And people do it if they maybe have an A level in the area but not a degree. 
+
+I personally didn’t have a degree in the area, so it felt quite intense and it was a lot of work, but I really loved doing it, and it’s allowed me to start my PGCE in physics with confidence that I know that I have the ability to teach that subject. 
+
+**Speaker 2: Michael Ashton, SKE Co-ordinator, Edge Hill University:** People perhaps have been in accountancy and think, well you know, “I would like to be a maths teacher” so there’s relevancy there. They might have a degree in maths but want to teach physics, so we would look at the whole person’s profile and then make a decision and recommend whether they needed subject knowledge enhancement. 
+
+**Speaker 3:** I did my A Level but it was nearly 9 years ago now – I'm getting on a bit! So I thought, even I knew that I’d need to polish up on my maths course and knowledge from A Level, just so I’ve got a bit of certainty when I teach it. 
+
+**Speaker 4:** Originally, I was interested in biology but when I contacted the University of Southampton, they said that I didn’t have enough to do biology but because I’d done A Level physics, the physics programme would be very much open to me to apply. And as a result of that, got more interested in that. 
+
+**Speaker 2: Michael Ashton, SKE Co-ordinator, Edge Hill University:** There’s lots of people who’ve thought about teaching, perhaps drifted into different career paths and they’ve perhaps had a hankering, a desire in the background to be a teacher. I think it’s particularly for those sort of people who’ve got in a job that’s bringing money in, but they’re not totally satisfied in what they’re doing and they think teaching will give them more job satisfaction – the idea of making a difference to people and the world. 
```

Comment:
Do we need to repeat his job title again here (or even his name?)? It feels a bit unnecessary and repetitive?